p2p/libp2p/netMessenger.go

```diff
@@ -957,11 +953,6 @@ func (netMes *networkMessenger) RegisterMessageProcessor(topic string, identifie
 }
 
 func (netMes *networkMessenger) registerOnPubSub(topic string, topicProcs *topicProcessors) error {
-	if topic == common.ConnectionTopic {
```

Comment:
the fix will work only after all nodes will make the transition to the new version